source/appModules/soffice.py

```diff
@@ -101,6 +107,47 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		if backgroundColor:
 			formatField['background-color']=colors.RGB.fromString(backgroundColor)
 
+		if offset == 0:
+			# Only include the list item prefix on the first line of the paragraph.
+			numbering = formatField.get("Numbering")
+			if numbering:
+				formatField["line-prefix"] = numbering.get("NumberingPrefix") or numbering.get("BulletChar")
+
+		return formatField
+
+	def _getFormatFieldAndOffsets(  # noqa: C901
+			self,
+			offset: int,
+			formatConfig: Optional[Dict],
+			calculateOffsets: bool = True
+	) -> Tuple[textInfos.FormatField, Tuple[int, int]]:
+		obj = self.obj
+		try:
+			startOffset, endOffset, attribsString = obj.IAccessibleTextObject.attributes(offset)
+		except COMError:
+			log.debugWarning("could not get attributes", exc_info=True)
+			return textInfos.FormatField(), (self._startOffset, self._endOffset)
+
+		if not attribsString and offset > 0:
+			try:
+				attribsString = obj.IAccessibleTextObject.attributes(offset - 1)[2]
+			except COMError:
+				pass
+
+		# LibreOffice >= 24.2 uses IAccessible2 text attributes, earlier versions use
+		# custom attributes, with the attrtibutes string starting with "Version:1;"

```

Comment:
attrtibutes -> attributes:

```suggestion
		# custom attributes, with the attributes string starting with "Version:1;"
```

Reply:
Fixed in the new version.